tests/view/observer/mutationobserver.js

```diff
@@ -267,6 +267,66 @@ describe( 'MutationObserver', () => {
 		expect( lastMutations[ 0 ].newText ).to.equal( 'xy' );
 	} );
 
+	// https://github.com/ckeditor/ckeditor5/issues/692 Scenario 1.
+	it( 'should handle space after inline filler at the end of container', () => {
+		const { view, selection } = parse( '<container:p>foo<attribute:b>[]</attribute:b></container:p>' );
+
+		viewRoot.appendChildren( view );
+		viewDocument.selection.setTo( selection );
+
+		viewDocument.render();
+
+		const inlineFiller = domEditor.childNodes[ 2 ].childNodes[ 1 ].childNodes[ 0 ];
```

Comment:
It'd be nice to comment what is in `domEditor.childNode[ 2 ]`. Here and below.

Reply:
Or how DOM looks like.

Reply:
Two children are appended in the `beforeEach()` to root. In these three tests, I'm only adding new elements to prevent some weird actions happening in the `MutationObserver`. Few tests for the inline filler does it. I'll add a note to these tests.